addons/playersuppression/functions/fnc_fired.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: LAxemann
+ * initial checks, gets config values etc, input from the fired EH
+ */
+#include "script_component.hpp"
+private ["_unit"];
+_unit		= _this select 0;         // Unit that shot the projectile
+if ((player distance _unit ) >= 2700) exitWith {};

```

Comment:
To explan:
ACE_player is the currently controlled avatar including zeus. It's the cornerstone of making everything zeus remote control compatible.